apps/projects/app/components/Panel/ReviewApplication/ReviewApplication.js

```diff
@@ -1,21 +1,157 @@
 import PropTypes from 'prop-types'
 import React from 'react'
+import styled from 'styled-components'
 
-const ReviewApplication = ({ issues, rate, onSubmit }) => (
-  <div>ReviewApplication</div>
-)
-
-ReviewApplication.propTypes = {
-  /** array of issues to allocate bounties on */
-  issues: PropTypes.arrayOf(
-    PropTypes.shape({
-      id,
-      level,
-    })
-  ),
-  /** base rate in pennies */
-  rate: PropTypes.number,
-  onSubmit: PropTypes.func,
+import {
+  Field,
+  Text,
+  TextInput,
+  Button,
+  Info,
+  SafeLink,
+} from '@aragon/ui'
+
+import { FormField, FieldTitle } from '../../Form'
+import { IconGitHub, CheckButton } from '../../Shared'
+
+// external data, all of it
+const application = {
+  user: {
+    login: 'jsnow',
+    name: 'Jon Snow',
+    avatar: 'https://avatars0.githubusercontent.com/u/34452131?v=4'
+  },
+  workplan: 'I solemnly swear to work on it day and night until it is done.',
+  hours: 13,
+  eta: '9/6/2018',
+}
+
+class ReviewApplication extends React.Component {
+  static propTypes = {
+    issue: PropTypes.object.isRequired
+  }
+
+  state = {
+    feedback: ''
+  }
+
+  changeField = ({ target: { name, value } }) => this.setState({ [name]: value })
+
+  onReviewApplicationAccept = () => {
+    console.log('Accepted', this.state.feedback, application)
+  }
+  onReviewApplicationReject = () => {
+    console.log('Rejected', this.state.feedback, application)
+  }
+
+  render() {
+    const { title, repo, number, url } = this.props.issue
+
+    return (
+      <div>
+        <IssueTitle>{title}</IssueTitle>
+        
+        <SafeLink
+          href={url}
+          target="_blank"
+          style={{ textDecoration: 'none', color: '#21AAE7' }}
+        >
+          <IssueLinkRow>
+            <IconGitHub color="#21AAE7" width='14px' height='14px' />
+            <Text style={{ marginLeft: '6px'}}>{repo} #{number}</Text>
+          </IssueLinkRow>
+        </SafeLink>
+
+        <ApplicationDetails>
+          <UserLink>
+            <img src={application.user.avatar} style={{ width: '32px', height: '32px', marginRight: '10px'}} />
+            <SafeLink
+              href={url}
+              target="_blank"
+              style={{ textDecoration: 'none', color: '#21AAE7', marginRight: '6px' }}
+            >
+              {application.user.name ? application.user.name : application.user.login}
+            </SafeLink>
+            applied 8 days ago
```

Comment:
Can we get this as a variable even if it's hardcoded for the time being?

Reply:
done.